end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -928,6 +934,8 @@ evaluated for merging.
   the lower intensity, are merged. Values for this parameter should not be too 
   small to avoid merging closely co-eluting ions, such as isomers. 
 
+(Dan: If present in the current dataset - and if we have such information - perhaps would be useful to also show some examples of wrong processing, like using too high of expandMz values and merging isotopes, or too small minProp and merging isomers.)
```

Comment:
I'll consider adding code for this,  I don't wanna lengthen much more the workflow, if too complicated we can at least mention it, you are very right !